lighthouse-core/fraggle-rock/config/default-config.js

```diff
@@ -11,12 +11,14 @@ const {deepClone} = require('../../config/config-helpers.js');
 /** @type {LH.Config.AuditJson[]} */
 const frAudits = [
```

Comment:
We can probably just add these audits + category extension to the default config directly. I think this workaround predates the `supportedModes` filter.

Reply:
> We can probably just add these audits + category extension to the default config directly. I think this workaround predates the `supportedModes` filter.

Do you mean the classic default config? Only the FR config will filter based on `supportedModes`.

If you mean the new default config, adding it to the audits array is fine but adding it to the perf category is awkward or needs adjusting. I'll just leave it as it is here and it can move when the new default config is refactored to not be based on the classic default config.

Reply:
> Do you mean the classic default config? Only the FR config will filter based on supportedModes.

Hmm yeah you're right. For some reason I thought classic configs would filter by gather mode. Anyway this is fine.